views/endorsement-backers.js

```diff
@@ -0,0 +1,103 @@
+const { html } = require('../helpers')
+const stateNames = require('datasets-us-states-names-abbr')
+
+const legislatorTitles = {
+  California: ['Senator', 'Assembly'],
+  'Dane County': ['Supervisor'],
+}
+
+module.exports = (state, dispatch) => {
+  const { backersFilterQuery, measures, loading, location, votes } = state
+  const vote = votes[location.params.voteId]
+  const backers = vote.backers
+  const filteredBackers = backers && backers.reduce((memo, backer, index) => (
+    passesFilter({ ...backer, index: index + 1 }, backersFilterQuery)
+      ? memo.concat({ ...backer, index: index + 1 })
+      : memo
+  ), [])
+  const measure = measures[location.params.shortId]
+  const numLegislators = backers && backers.reduce((max, backer) => Math.max(max, backer.offices.length), 0)
+  const titles = legislatorTitles[measure.legislature_name] || []
+
+  return html`
+    <div>
+      ${loading.backers || !filteredBackers ? html`
+        <h3>Loading backers...</h3>
+      ` : html`
+        <div>
+          ${searchBar(dispatch, { backers, backersFilterQuery, filteredBackers })}
+          <table class="table is-narrow is-bordered is-striped" style="display: block; overflow-x: auto; max-width: 858px; border-right: 1px solid #e6e6e6">
+            <thead>
+              <tr>
+                <th></th>
+                <th>Time</th>
+                <th>Name</th>
+                <th>Location</th>
+                ${numLegislators > 0 ? html`<th>${titles[0] || 'Legislator'}</th>` : ''}
+                ${numLegislators > 1 ? html`<th>${titles[1] || 'Legislator 2'}</th>` : ''}
+                <th>Comment</th>
+              </tr>
+            </thead>
+            <tbody>
+              ${filteredBackers.map((b) => backersTableRow(b, numLegislators))}
+            </tbody>
+          </table>
+        </div>
+      `}
+    </div>
+  `
+}
+
+const last = (array) => array[array.length - 1]
+
+const backersTableRow = (backer, numLegislators) => {
+  const reps = backer.offices.map(office => {
+    if (!office) { return '' }
+    const { office_holder } = office
+    if (!office_holder) { return '' }
+    return `${office_holder.first_name} ${office_holder.last_name} (D-${last(office.name.split(' '))})`
+  })
+  const name = backer.public ? backer.name : '[private]'
+  const stateAbbr = stateNames[backer.administrative_area_level_1]
+
+  return html`
+    <tr>
+      <td><span style="width: 30px; display: inline-block; text-align: right;">${backer.index}</span></td>
+      <td><span style="width: 165px; display: inline-block;">${new Date(backer.created_at).toLocaleString()}</span></td>
+      <td><span style="width: 165px; display: inline-block;">${name}</span></td>
+      <td><span style="width: 145px; display: inline-block;">${backer.locality}${backer.locality && stateAbbr ? `, ` : ''}${stateAbbr}</span></td>
+      ${numLegislators > 0 ? html`
```

Comment:
Another instance of hardcoded reps.

Reply:
Fixed by https://github.com/voteliquid/liquid.us/pull/155/commits/e2ab5510634242ca787ddc3799be788095f33d8e